### ISSUE TITLE
Idris can't find Prelude.idr unless run from home directory

### ISSUE THREAD

**Issue by bishboria** [idris-lang/Idris-dev#160]
I'm getting an odd behaviour on idris-0.9.5.1

run idris from the folder where it was "cabal-dev install"ed

```
$ idris
/Users/bishboria/Documents/Programming/idris/idris-tutorial/cabal-dev//share/base/Prelude.idr: openFile: does not exist (No such file or directory)
```

(not sure where it's picking up the extra "/" before share...)

cd back to my home directory and try again and I don't get that problem...

In order to run/compile files that use anything from Prelude I have to move them to my home directory first. Is this related to https://github.com/edwinb/Idris-dev/issues/81 ?

My setup, if it helps: OS X 10.7.5, ghc platform 2012.4.0.0 (64bit), cabal-dev 0.9.1 (built with Cabal 1.16.0.3)

**Comment by tobia** [comment]
I thought I was getting this same error with Idris 1.1.1 on Linux. From my home directory Idris worked, but if I ran it from the directory where I keep my tests it gave this weird error:

```
/home/tobia/.cabal/share/x86_64-linux-ghc-7.10.3/idris-1.1.1/libs/prelude/Prelude.idr: openFile: does not exist (No such file or directory)
```

But it all worked fine until a couple days ago. So I used one of my favorite tools, `strace`, and found the culprit: I had named one of my tests `IO.idr` and this was making Idris barf with this weird error.

I don't know if it's a bug or expected behavior. I'm just leaving it here in case others give a file a bad name and end up here through Google.
